site/content/docs/5.0/components/alerts.md

```diff
@@ -147,35 +147,41 @@ Loop that generates the modifier classes with the `alert-variant()` mixin.
 
 ## JavaScript behavior
 
-### Triggers
+### Initialize
 
-Enable dismissal of an alert via JavaScript:
+Initialize elements as alerts
 
 ```js
 var alertList = document.querySelectorAll('.alert')
-alertList.forEach(function (alert) {
-  new bootstrap.Alert(alert)
+var alerts = [...alertList].map(function (el) {
```

Comment:
I wouldn't use spread yet. The docs are everywhere ES5

Reply:
I'm so used to them already, I didn't even recognize that. Good catch :+1: